builder/parser/line_parsers.go

```diff
@@ -107,12 +107,15 @@ func parseNameVal(rest string, key string) (*Node, map[string]bool, error) {
 				quote = ch
 				blankOK = true
 				phase = inQuote
-				continue
```

Comment:
@duglin it's not clear to me anymore if it's this function's or the shellParser's responsibility to handle quotes.